### DISCUSSION TITLE
Looking for admin password???

### DISCUSSION THREAD

**Discussion by cooley420** [GerryDazoo/Slinger#600] [General]
If you are in need of your admin password try the following.   Download the slingplayer desktop app to your computer.  Open the app and sign in with your email and password.  The slingbox servers are long gone so it will never connect, but give it a few minutes and then search your computer for "~/appdata/roaming/sling media/slingplayer desktop/logs/console.log".  Once this file is created, it took a few minutes for me, you can then open it, and search for "password", toward the bottom of the file you will find "USER":"admin","PASSWORD":"XXXXXXXXXXXXX" where XXXX is your 15 digit password.  Good luck everyone.

**Comment by hlyl** [comment]
Not able to get the version of the slingplayer programme that i was able to find to start on my windows 11 machine. Have tried compability with windows vista but not luck

**Comment by flipper2222222** [comment]
which version of sling player are you after?

**Comment by midcorestudios** [comment]
I've tried 5.0.0.83 and it just sits on the splash screen, I can no longer get to the sign in page, i've tried all previous versions of 5 as well with the same results.  I was able to open it about 2 weeks ago and get a sign in page.

**Reply by GerryDazoo** [reply]
Have you inadvertently blocked access to the sling servers from your Windows box?

**Reply by midcorestudios** [reply]
yea i even tried reinstall windows, and running in a vm.  Also tried on a mac and same stuck on the splash screen.  did a factory reset of my modem as well just in case something there was blocking it.

**Reply by skcuslleb** [reply]
Resurrecting this as I'm having the same problem. Were you able to find a solution?

**Comment by flipper2222222** [comment]
this link has 2 versions of the slingplayer I used 1.5.1.338 to unbrick my solo hp pro
https://www.reddit.com/r/slingbox/comments/ymvtkf/config_inputs_future_proof_idea_thanks_for_the/
